submissions/mustbefail/dom-api-task/script.js

```diff
@@ -0,0 +1,56 @@
+const getMoviesByGenre = async (movieGenre) => {
```

Comment:
This is a very big functional feature!
Try to break it down into smaller, goal-oriented functions.
Such as create, load, render